server/listeners/eventHandlers/probationEvents/offenderManagerChangedEventHandler.ts

```diff
@@ -1,15 +1,11 @@
-import { ProbationEvent } from '../../../@types/events'
+import { ProbationEventMessage } from '../../../@types/events'
 import CommunityService from '../../../services/communityService'
 import LicenceService from '../../../services/licenceService'
 
 export default class OffenderManagerChangedEventHandler {
   constructor(private readonly communityService: CommunityService, private readonly licenceService: LicenceService) {}
 
-  // TODO: I suspect this is an incorrect interpretation of the meaning of this event?
```

Comment:
Think we have agreed that this listener is doing as it should, and another listener would be added to handle the OFFICER_DETAIL_CHANGED event

Reply:
Yes, agreed.